datanode/api/trading_data_v2.go

```diff
@@ -2434,10 +2433,12 @@ func (t *TradingDataServiceV2) GetEpoch(ctx context.Context, req *v2.GetEpochReq
 		epoch entities.Epoch
 		err   error
 	)
-	if req.GetId() == 0 {
-		epoch, err = t.epochService.GetCurrent(ctx)
+	if req.GetId() > 0 {
```

Comment:
Should we just make it an error if a block and an ID are given at the same time?

Reply:
We could. Right now we just ignore the block if id was provided. I guess returning an error would let the client know that they don't have to bother with both params. On the other hand it's not really hurting anyone

Reply:
Sure. Can you make sure that behaviour is documented in the protos docstring somewhere.